src/community/app-schema/webservice-test/src/test/java/org/geoserver/test/WebServiceBackendWfsTest.java

```diff
@@ -1,711 +0,0 @@
-/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
- * (c) 2001 - 2013 OpenPlans
- * This code is licensed under the GPL 2.0 license, available at the root
- * application directory.
- */
-
-package org.geoserver.test;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
-
-import java.io.IOException;
-import java.util.ArrayList;
-
-import org.geoserver.wfs.xml.v1_1_0.WFS;
-import org.geotools.data.complex.AppSchemaDataAccess;
-import org.junit.Test;
-import org.w3c.dom.Document;
-
-/**
- * WFS GetFeature to test integration of {@link AppSchemaDataAccess} with web service backend with
- * GeoServer.
- *
- * @author Rini Angreani, CSIRO Earth Science and Resource Engineering
- */
-public class WebServiceBackendWfsTest extends AbstractAppSchemaTestSupport {
```

Comment:
Why was this test removed? Should not be in this PR in any case

Reply:
The file was accidentally formated and commited and has nothing to do with the WebP module. What is the right way to remove this from the PR?